lessons/codap-bar-charts/langs/en-us/pages/codap-displays-intro.adoc

```diff
@@ -1,85 +1,59 @@
-= Data Displays in CODAP
+= Dot Plots and Bar Charts in CODAP
 
-Make sure you’re logged into CODAP, with the Animals Dataset open. Create a graph of randomly generated points by selecting the `Graph` icon, and then respond to the following prompts.
+Open the @starter-file{animals}. First, create a graph of randomly generated points by selecting the `Graph` icon, and then respond to the following prompts.
 
-== Creating Bar Charts in CODAP
+== Create Displays
 
-@n Select the y-axis on your graph (where it says "Click here"). On the drop-down menu that apperas, select `fixed`. (_If you prefer, you may also drag the attribute name from the table to the y-axis._) What do you notice?
+@n Select the y-axis on your graph (where it says "Click here"). On the drop-down menu that appears, select `Fixed`. (_If you prefer, you may also drag the attribute name from the table to the y-axis._) What do you notice?
 
-
-@ifsoln{A dot plot forms, categorizing animals as fixed (true) and not fixed (false). The labels for the dot plot are on the y-axis. The dots are stacked horizontally.}
-
-@ifnotsoln{
-@fitb{}{}
-
-@fitb{}{}
+@fitb{}{@ifsoln{A horizontal dot plot forms, categorizing animals as fixed (true) and not fixed (false). The labels for the dot plot are on the y-axis.}}
 
 @fitb{}{}
-}
 
 @vspace{1ex}
 
-@n Now select the x-axis on your graph and select `fixed`. How does the graph change?
+@n Now select the x-axis on your graph and select `Fixed`. How does the graph change?
 
-@ifsoln{The labels move from the y-axis to the x-axis, and the dots are stacked vertically.}
+@fitb{}{@ifsoln{The labels move from the y-axis to the x-axis, and the dots are stacked vertically.}}
 
-@ifnotsoln{
 @fitb{}{}
 
-@fitb{}{}
-}
 @vspace{1ex}
 
 @n Select the `configuration` icon (which looks like a bar graph) to the right of the data display. Select `fuse dots into bars`
 
 @n Click the `ruler` icon, and test out each of the two options available (count and percentage). What happens?
```

Comment:
There are now 3 options (the third being formula). Do you want to acknowledge all 3 or change the wording so that we're just focusing on these two rather than the available options?